### PR TITLE
Initial document commit

### PR DESCRIPTION
**Pull request by JonathanMcCaffrey**
General introductory maintenance documents

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+
+
+## Run the mock API.
+
+### ./
+```shell
+ amplify mock api
+```
+
+Open the [mock API](http://192.168.12.162:20002/).
```
Comment (DanielRDias) [inline docs/getting-started.md:134]: Can't we use 127.0.0.1 or localhost? `192.168.12.162` might not be valid to everyone

File: docs/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+Click **[Generate Token]**.
+
+Your future mock GraphQL calls will use this created auth instance automatically.
+
+
+## Create the Mock Data
+
+Enter the following text into the GraphiQL Editor and run the command.
+
+### http://192.168.12.162:20002/
```
Comment (DanielRDias) [inline docs/getting-started.md:149]: Same as previous comment

File: docs/getting-started.md (hunk 1 above)
Comment (JonathanMcCaffrey) [reply docs/getting-started.md]: Thanks, for the catch. Fixed

File: docs/getting-started.md (hunk 2 above)
Comment (JonathanMcCaffrey) [reply docs/getting-started.md]: Fixed

**Review by DanielRDias — APPROVED** [review]

**Pull request closed without merging**